README.md

```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
```

Comment:
I also installed the Command Line Tools, while I was trying to figure out how to build libcurl.  I think I remember you telling me that I needed to install something with XCode that I didn't have, but I don't remember what it was.  After the curl got eliminated, the project just built, and I didn't need to install anything else, so I'm wondering if the Command Line Tools were a requirement.